frontend/package.json

```diff
@@ -16,6 +16,7 @@
     "lint-staged": "^12.3.7",
     "react": "^18.0.0",
     "react-dom": "^18.0.0",
+    "react-hook-form": "^7.30.0",
```

Comment:
Ok, new library. We should discuss more about this decision, but I think it's a good one lib.